docs/development/building/Building-in-Fedora.md

```
@@ -1,41 +1,98 @@
+title: Building in Fedora
+---
+
+# Building in Fedora 44
+
+This guide covers building the Betaflight firmware and configurator on Fedora 44.
+
+### Clone Betaflight Repository and Install Toolchain
+
+Install build dependencies (`clang` and `libblocksruntime-devel` are needed for unit tests via `make test`):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)